docker-compose-pgsql.yml

```diff
@@ -34,6 +34,8 @@ services:
 
   db:
     image: postgres:alpine
+#    ports:
```

Comment:
Why?

Reply:
i need to extract it as development modification, otherwise it does not load the server in the default port of postgres - i will remove it.

Reply:
Then please revert and don't commit what you only need changed in your local environment.

Reply:
this is exactly what I did, this is written in my comment :)